README.md

```diff
@@ -24,6 +24,8 @@ biogeochemistry, is appended to the name.
 
 Currently the following configurations are available:
 
+FIXME: Will this be changed to `release-<config>`?
+
 - [`1deg_jra55do_ryf`](https://github.com/ACCESS-NRI/access-om3-configs/tree/1deg_jra55do_ryf)
 - [`1deg_jra55do_iaf`](https://github.com/ACCESS-NRI/access-om3-configs/tree/1deg_jra55do_iaf)
 - [`1deg_jra55do_ryf_wombatlite`](https://github.com/ACCESS-NRI/access-om3-configs/tree/1deg_jra55do_ryf_wombatlite)
```

Comment:
Also:
- [`dev-025deg_jra55do_ryf`](https://github.com/ACCESS-NRI/access-om3-configs/tree/dev-025deg_jra55do_ryf)

Reply:
Updated in https://github.com/ACCESS-NRI/access-om3-configs/pull/122/commits/db99e671d499d819b3423db87b30156bb675453b